src/python/WMCore/DataStructs/Run.py

```diff
@@ -101,6 +101,9 @@ def lumis(self):
     def lumis(self, lumiList):
         self.eventsPerLumi = list(lumiList)
 
+    def extendLumis(self, lumiList):
```

Comment:
@vkuznet for a further refactoring, I'm trying to turn lumis into a property rather than a list. But we have a few places that treat it as a list trying to directly extend or append to it.

Do you know of a way to automatically attach a method to the property so that I don't have to modify the other code. In other words, I want to still use myRun.lumis.append(42)